jjb/edgex-templates-docker.yaml

```diff
@@ -166,3 +166,52 @@
           properties-file: 'env_inject.txt'
       # Execute any post build clean-up or testing
       - shell: '{post_build_script}'
+
+- job-template:
+    name: '{project-name}-{stream}-release-version-docker-daily-no-sonar'
+    # Job template for Docker daily release jobs
+    #
+    # The purpose of this job template is to run "docker build and docker push"
+    # for projects using this template.
+    #
+    # Required Variables:
+    #     branch:    git branch (eg. stable/lithium or master)
+    <<: *docker_job_boiler_plate
+
+    docker_registry: '$DOCKER_REGISTRY:10004'
```

Comment:
Just a comment really, not a requested change (at this time) we should probably add some extra Jenkins ENV vars like DOCKER_STAGING_PORT DOCKER_SNAPSHOT_PORT and DOCKER_RELEASE_PORT so that these templates can be more easily moved to global-jjb and sort of "just work"